src/rollup/types.d.ts

```diff
@@ -241,17 +243,8 @@ export interface InputOptions {
 	chunkGroupingSize?: number;
 	shimMissingExports?: boolean;
 
-	// undocumented?
-	pureExternalModules?: boolean;
-	preferConst?: boolean;
-	perf?: boolean;
-
 	/** @deprecated */
-	entry?: string;
-	transform?: TransformHook;
-	load?: LoadHook;
-	resolveId?: ResolveIdHook;
-	resolveExternal?: any;
+	preferConst?: boolean;
```

Comment:
I know this one is not documented but I am not sure it should be deprecated. Maybe it should be documented instead? `const` seems to have a slight performance benefit in certain environments while `var` has the universal compatibility.

Reply:
Yes agreed.